src/index.ts

```diff
@@ -65,11 +66,7 @@ const resolveDate = (d: { dateTime?: string; format?: string }) => {
     return "All Day";
   }
   const date = new Date(d.dateTime);
-  if (d.format) {
-    return format(date, d.format);
-  } else {
-    return date.toLocaleTimeString();
-  }
+  return format(date, d?.format ? d.format : DEFAULT_DATE_FORMAT);
```

Comment:
another nice shorthand for this is `d?.format || DEFAULT_DATE_FORMAT`

The OR operation in javascript is a useful way to say "use this first value if it's defined, and if not, used the second one"

Reply:
Noted, will replace in my upcoming PR 💥